template2.css

```diff
@@ -0,0 +1,98 @@
+* {
```

Comment:
margin and padding always default set 0px

Reply:
default not zero